src/app/reporting/reporting.cpp

```diff
@@ -15,9 +15,11 @@
  *    limitations under the License.
  */
 #include "reporting.h"
+#include "lib/support/CodeUtils.h"
```

Comment:
I'm not i understand why sometimes we use `""` and other `<>`. Should we aim to only use `<>`?

Reply:
My editor auto-adds these and I miss converting them. `<>` is the correct thing to do I believe. Sorry I missed some.